sympy/physics/control/routh_table.py

```diff
@@ -0,0 +1,238 @@
+from sympy.matrices.dense import MutableDenseMatrix
+from sympy import Poly
+from sympy import cancel
+from sympy import symbols, Symbol
+
+__all__ = ['RouthHurwitz']
+
+
+class RouthHurwitz(MutableDenseMatrix):
+    r"""
+    A class for creating a Routh-Hurwitz table from a given polynomial.
+    It handle special cases with methods discussed in [1].
+
+    Note: When a row of the table is zero, the property ``zero_row_case`` is set to True.
+
+    Explanation
+    ============
+
+    In mathematics, the Routh-Hurwitz table is used to determine the number of roots of a polynomial that have positive or negative real parts.
+
+    It's crucial in the control system theory because it can be used to retrieve
+    necessary and sufficient conditions for the stability of a linear time-invariant control system.
+
+    Once the table is constructed, the stability of the system can be assessed by counting the number of sign changes in the first column.
+    Each sign change corresponds to a root with a positive real part, whereas each preservation of sign corresponds to a root with a negative real part.
+
+    There are two special cases to consider during the construction of the table:
+
+    1.  First Column Zero Case:
+        If a zero appears in the first column of a row (while the row is not entirely zero), the zero is replaced with a small symbolic infinitesimal ($\epsilon$, by default).
+        When checking for sign changes, the limit $\epsilon \to 0$ is taken.
```

Comment:
```suggestion
        When checking for sign changes, the limit :math:`\epsilon \to 0` is taken.
```

Using the `$...$` means math will only display on the sympy documentation website, but in no other tools that render RST files. `:math:` directive is preferred.